### PR TITLE
[ru] Added Russian lang

### PR DESCRIPTION
**Pull request by FlymeDllVa**
Signed-off-by: Dmitry Gridnev <flymedllva@gmail.com>

### CODE REVIEW

**Comment by bridgetkromhout** [comment]
Hi, @FlymeDllVa - thank you! This is great. Can you please sign the DCO so that we can merge this PR? Here is the info for how: https://github.com/helm/helm-www/pull/1029/checks?check_run_id=1973124836

File: content/ru/docs/intro/_index.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+title: "Введение"
+weight: 1
+aliases: ["/docs/using_helm/"]
```
Comment (bridgetkromhout) [inline content/ru/docs/intro/_index.md:4]: You can remove all the aliases lines. Those are only necessary for English docs that got moved to a different location. They don't apply to new translations.

File: content/ru/docs/intro/install.md [hunk 2]
```diff
@@ -0,0 +1,152 @@
+---
+title: "Установка Helm"
+description: "Узнайте, как установить и начать работать с Helm."
+weight: 2
+aliases: ["/docs/install/"]
```
Comment (bridgetkromhout) [inline content/ru/docs/intro/install.md:5]: You can remove all the aliases lines. Those are only necessary for English docs that got moved to a different location. They don't apply to new translations.

File: content/ru/docs/intro/quickstart.md [hunk 3]
```diff
@@ -0,0 +1,127 @@
+---
+title: "Краткое Руководство"
+description: "How to install and get started with Helm including instructions for distros, FAQs, and plugins."
+weight: 1
+aliases: ["/docs/quickstart/"]
```
Comment (bridgetkromhout) [inline content/ru/docs/intro/quickstart.md:5]: You can remove all the aliases lines. Those are only necessary for English docs that got moved to a different location. They don't apply to new translations.

**Comment by bridgetkromhout** [comment]
It looks like we also need the entry in https://github.com/helm/helm-www/blob/master/config.toml as explained in https://helm.sh/docs/community/localization/#navigating-between-languages .

**Comment by FlymeDllVa** [comment]
> It looks like we also need the entry in https://github.com/helm/helm-www/blob/master/config.toml as explained in https://helm.sh/docs/community/localization/#navigating-between-languages .

Снимок экрана 2021-02-28 в 12 29 01

File: content/ru/docs/intro/using_helm.md [hunk 4]
```diff
@@ -0,0 +1,482 @@
...
+
+Теперь вы можете с легкостью поставить этот chart в ваш Helm с помощью `helm install`
+
+```console
+$ helm install deis-workflow ./deis-workflow-0.1.0.tgz
+...
+```
+
+Упакованные chart-ы могут быть загружены в репозиторий chart-ов. Смотрите документацию по
+[Helm chart репозитории](https://helm.sh/docs/topics/charts/) для того, что бы узнать новые детали.
```
Comment (FlymeDllVa) [inline content/ru/docs/intro/using_helm.md:474]: How can I do"ref" without a direct link? The page with the Russian translation is not yet available

File: content/ru/docs/intro/using_helm.md [hunk 5]
```diff
@@ -0,0 +1,482 @@
...
+```
+
+Поскольку репозитории диаграмм часто меняются, в любой момент вы можете убедиться,
+что ваш клиент Helm обновлен, запустив `helm repo update`.
+
+Репозитории могут быть удалены с помощью `helm repo remove`.
+
+## Создание Собственных Chart-ов
+
+[Руководство по разработке Chart-ов](https://helm.sh/docs/topics/charts/) объясняет, как
```
Comment (FlymeDllVa) [inline content/ru/docs/intro/using_helm.md:442]: How can I do"ref" without a direct link? The page with the Russian translation is not yet available

File: content/ru/docs/intro/using_helm.md (hunk 4 above)
Comment (bridgetkromhout) [reply content/ru/docs/intro/using_helm.md:474]: You either can add a copy of the untranslated page under `ru` (this is okay) or follow the suggestions in https://helm.sh/docs/community/localization/#resolving-internal-links to change the links.

**Review by bridgetkromhout — APPROVED** [review]
Thanks, @FlymeDllVa! I fixed those few links to make them point at the English pages until the Russian ones exist. The site builds correctly now, and I'll merge this. Thank you!

**Pull request merged**